### PR TITLE
Fix #275386: anchor points not rebuilt after setting autoplacement

### PR DESCRIPTION
**Pull request by jthistle**
Addresses: https://musescore.org/en/node/275386

### CODE REVIEW

File: mscore/scoreview.h [hunk 1]
```diff
@@ -397,6 +396,8 @@ class ScoreView : public QWidget, public MuseScoreView {
       void changeState(ViewState);
 
       virtual const QRect geometry() const override { return QWidget::geometry(); }
+
+      void updateGrips();
```
Comment (anatoly-os) [inline mscore/scoreview.h]: Why did you move it? Was it intentionally?

Comment (jthistle) [reply mscore/scoreview.h]: Yes, I moved it so that it would be a public method rather than a private one.

**Comment by anatoly-os** [comment]
@jthistle could you please rebase the code so we could merge the PR.

**Comment by jthistle** [comment]
Done, just waiting for tests now @anatoly-os
